test-frame-metrics-collector/src/main/java/io/skodjob/testframe/MetricsCollector.java

```diff
@@ -440,11 +440,13 @@ public final void collectMetricsFromPods(long timeoutDuration) {
                         if (metricsData.isEmpty()) {
                             status.setMessage("No pods found or no metrics available from pods.");
                             status.setType(MetricsCollectionStatus.Type.NO_DATA);
+                            LOGGER.error("Metrics collection failed: {}", status.getMessage());
```

Comment:
I assume that the collecting will be tried again so maybe use just warn for these kind of problems? And keep the error log just on timeout?

Reply:
Yes it seems reasonable thanks :)